core/src/test/kotlin/com/bumble/appyx/core/lifecycle/MinimumCombinedLifecycleTest.kt

```diff
@@ -40,9 +40,13 @@ class MinimumCombinedLifecycleTest {
     @Test
     fun `destroyed when one of lifecycles is destroyed`() {
         val test1 = TestLifecycle()
-        test1.state = State.DESTROYED
+        test1.state = State.CREATED
```

Comment:
This is due to updating the Lifecycle library (unrelated to this change). I'm unsure if this is the correct way to fix this, but it wouldn't be merged in this PR (it would be in a different PR).

Without this change, the test fails with: `no event down from INITIALIZED` because they don't want you to immediate move to the DESTROYED state.